tools/packaging/common/istio-iptables.sh

```diff
@@ -223,9 +223,11 @@ fi
 # need to split them in different arrays one for ipv4 and one for ipv6
 # in order to not to fail
 pl='/*'
+IFS=',' read -ra EXCLUDE <<< "$OUTBOUND_IP_RANGES_EXCLUDE"
+IFS=',' read -ra INCLUDE <<< "$OUTBOUND_IP_RANGES_INCLUDE"
```

Comment:
I don't really have a clue what this does. not saying it is wrong, I see this in a lot of places so it is probably right, but I can't give a good review.

Maybe we need to port the tests they added in the distroless PR so we can get some confidence on these fixes?

Reply:
It converts a comma separated string into an array so then each element of this string can be tested individually.

Reply:
@howardjohn -- could you point at the tests you're referring to?  this pr #12869?

Reply:
@howardjohn @tiswanso I modified e2e-simple to test parsing of the list of exclude ranges, so this part of code will be tested by a required test.